tests/test_std.rs

```diff
@@ -909,6 +909,21 @@ fn combinations_zero() {
     it::assert_equal((0..0).combinations(0), vec![vec![]]);
 }
 
+#[test]
+fn combinations_range_count() {
+    for n in 0..6 {
+        for k in 0..=n {
+            let len = (n - k + 1..=n).product::<usize>() / (1..=k).product::<usize>();
```

Comment:
Please `assert_eq(len, it.count)` and `assert_eq(len, it.size_hint().0)` and `assert_eq(Some(len), it.size_hint().1)`.